dcsazure_adls_to_adls_mask_pl/README.md

```diff
@@ -0,0 +1,75 @@
+# dcsazure_adls_to_adls_mask_pl
+## Delphix Compliance Services (DCS) for Azure - ADLS to ADLS Masking Pipeline
+
+This pipeline will perform masking of data from your Azure Data Lake (ADLS) Data from one location to another.
+
+### Prerequisites
+
+1. Configure the hosted metadata database and associated Azure SQL service (version `V2024.04.18.0`+).
+1. Configure the DCS for Azure REST service.
+1. Configure the Azure Data Lake Storage service associated with your ADLS source data.
+1. Configure the Azure Data Lake Storage service associated with your ADLS sink data.
+
+
+### Importing
+There are several linked services that will need to be selected in order to perform the masking of your ADLS
+instance.
+
+These linked services types are needed for the following steps:
+
+`Azure Data Lake Storage` (source) - Linked service associated with ADLS source data. This will be used for the
+following steps:
+* Switch On Copy Methodology And Supported Write (Switch activity)
+* dcsazure_adls_to_adls_mask_df/adlsSource (dataFlow)
+* dcsazure_adls_to_adls_copy_df/adlsSource (dataFlow)
+
+`Azure Data Lake Storage` (sink) - Linked service associated with ADLS sink data. This will be used for the
+following steps:
+* dcsazure_adls_to_adls_mask_df/adlsSink (dataFlow)
+* dcsazure_adls_to_adls_copy_df/adlsSink (dataFlow)
+
+`Azure ADLS ` (source) - Linked service associated with ADLS. This will be used for the
+following steps:
+* dcsazure_adls_to_adls_for_mask_query_ds (Azure adls  dataset)
+* dcsazure_adls_to_adls_copy_ds (Azure adls  dataset)
+
+`Azure SQL` (metadata) - Linked service associated with your hosted metadata store. This will be used for the following
+steps:
+* dcsazure_adls_to_adls_metadata_mask_ds (Azure SQL Database dataset)
+* dcsazure_adls_to_adls_mask_params_df/Ruleset (dataFlow)
+* dcsazure_adls_to_adls_mask_params_df/TypeMapping (dataFlow)
+
+`REST` (DCS for Azure) - Linked service associated with calling DCS for Azure. This will be used for the following
+steps:
+* dcsazure_adls_to_adls_mask_df (dataFlow)
+
+### How It Works
+* Select Tables That Require Masking
+    * Choose, based on what is present in the metadata data store, which tables require masking
+      * For each table that requires masking:
+        * Call the `dcsazure_adls_to_adls_mask_params_df` data flow to generate masking parameters
+        * Call the `dcsazure_adls_to_adls_mask_df` data flow, passing in parameters as generated by
+          the generate masking parameters dataflow
+* Select Tables Without Required Masking
+    * Choose, based on what is present in the metadata data store, which tables do not require masking
+      * For each table that does not require masking, perform a copy activity.
+
+### Notes
+Sometimes the number of batches output in the `Get Mask Parameters` step will be incorrect. This is because ADLS does
+not support us storing the number of rows in the files, which means that we may run into request size limits. To work
```

Comment:
This line is confusing to me:
> This is because ADLS does not support us storing the number of rows in the files, which means that we may run into request size limits

Maybe something like:
>  This is because ADLS does not provide us the number of rows in the files, which means that we may hit request size limits.

Reply:
Fixed! Thanks